src/SmartComponents/BaselinesPage/EditBaselinePage/EditBaselinePage.js

```diff
@@ -284,24 +285,29 @@ export class EditBaselinePage extends Component {
     /*eslint-disable*/
     render() {
         return (
-            <PermissionContext.Consumer>
-                { value =>
-                    <React.Fragment>
-                        { this.renderPageHeader(value.permissions) }
-                        <Main>
-                            { value.permissions.baselinesRead === false
-                                ? <EmptyStateDisplay
-                                    icon={ LockIcon }
-                                    color='#6a6e73'
-                                    title={ 'You do not have access to view this Baseline' }
-                                    text={ [ 'Contact your organization administrator(s) for more information.' ] }
-                                />
-                                : this.renderMain(value.permissions)
-                            }
-                        </Main>
-                    </React.Fragment>
-                }
-            </PermissionContext.Consumer>
+            <RegistryContext.Consumer>
```

Comment:
I don't think there is a way to reach this in a fashion that would cause the issue described in the jira. 
But just to be sure.